sympy/simplify/tests/test_radsimp.py

```diff
@@ -393,6 +394,8 @@ def test_fraction():
     assert fraction(exp(n)) == (1, exp(-n))
     assert fraction(exp(-n)) == (exp(-n), 1)
 
+    p = symbols('p', positive=True)
+    assert fraction(exp(-p)*log(p), exact=True) == (exp(-p)*log(p), 1)
```

Comment:
If this change is made then the docstring should be updated. It says, 

```
If we know nothing about sign of some exponent and 'exact'
flag is unset, then structure this exponent's structure will
be analyzed and pretty fraction will be returned:
```

But in this case we *do* know something about the sign of the exponent. My read of the current docstring is that `exact` only applies to vanilla exponents.